eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentEvents.scala

```diff
@@ -35,19 +35,41 @@ sealed trait PaymentEvent {
   val timestamp: Long
 }
 
-case class PaymentSent(id: UUID, paymentHash: ByteVector32, paymentPreimage: ByteVector32, parts: Seq[PaymentSent.PartialPayment]) extends PaymentEvent {
-  require(parts.nonEmpty, "must have at least one subpayment")
-  val amount: MilliSatoshi = parts.map(_.amount).sum
-  val feesPaid: MilliSatoshi = parts.map(_.feesPaid).sum
+/**
+ * A payment was successfully sent and fulfilled.
+ *
+ * @param id              id of the whole payment attempt (if using multi-part, there will be multiple parts, each with
+ *                        a different id).
+ * @param paymentHash     payment hash.
+ * @param paymentPreimage payment preimage (proof of payment).
+ * @param finalAmount     amount that has been received by the final recipient.
```

Comment:
Maybe`recipientAmount` / `recipientNodeId` or `finalAmount` / `finalNodeId` for consistency?